Certification standard/main.html

```diff
@@ -0,0 +1,221 @@
+
+      <section id='certlevels-l1'>
+        <h3>Level 1: Automated testing</h3> 
+
+        Certification at level 1 is intended for verification of a <a>DApp</a> that can be performed by an unguided automated tool. The tool shall be able to determine if the code meets its requirements and is not vulnerable to some vulnerabilities [Discussion: Should we relate to the list of vulnerabilities and define the ones that can/should be dealt at this level, because the tools can do it].
+
+        [Todo (or not but tbd):
+        - Vulnerabilities to be checked
+        - Metrics to be reached
+        - Tracability to documentation
+        - List of tools/techniques? Fuzzing, mutation testing, concolic testing, symbolic execution, property based testing, etc.?
+        - Tool qualification
+        - [Others?]
+        ]
+      </section>
+
+      <section id='certlevels-l2'>
+        <h3>Level 2: Manual audit</h3>
+
+        Certification at level 2 is intended for a <a>DApp</a> that has been carefully reviewed by an auditor, through a manual analysis with possibly the use of automated tools. [Discussion: How de we split this from auditor doing only the work of level 1?]
+
+        Say something about CIP 52.
+
+      </section>
+
+      <section id='certlevels-l3'>
+        <h3>Level 3: Formal verification</h3>
+
+        Certification at level 3 is intended for the verification of a <a>DApp</a> that went through formal verification.
+        
+        [Todo (or not but tbd):
+        - What needs to be proven?
+        - Should it really be its own level? 
+        - Who reviews the proof? The assumptions? The theorems?
+        - Tool qualification?
+        - Tracability paper spec/formalized spec/model
+        - [others]
+        ]
+      </section>
+
+
+      <section>
+        <h3> Good practices </h3>
+
+        This section describes good practices 
+
+        [Todo (or not but tbd):
+        - Check new vulnerabilities discovered? Where?
+        - Disclose new vulnerabilities? How?
+        - Use latest stable compilers, testing tools, solvers? etc. 
+        - Open source code?
+        - Readable code?
+        - Documentation? Unless it's mandatory in the levels definition
+        - Coding standard? 
+        - [others?]
+        ]
+
+      </section>
+    </section>
+
+    <section id='tool-qualification'>
+      <h2 data-status="candidate"> Tool qualification</h2>
+
+        This section describes XXX about the tool used for verification, compilation. 
+
+        [TODO:
+          - Do we want something about tool qualification?
+          - Which tools should be in this considerations?
+            - Test generation tools?
+            - Testing frameworks?
+            - Other verification tools?
+            - Emulators?
+            - Compilers?
+            - ?
+          - Do we talk about versions?
+          - Do we point to an approved list? A checklist to be approved?
```

Comment:
I'd suggest a list without approvals. Is the list is to be approved, then who plays this role, and what sort of guarantee could they be said to be providing? One **process** idea: we could explicitly mark some sections TBD or version 2, so that we make clear that they have been thought of, but that discussion will take place later.